galaxy_ng/app/access_control/statements/roles.py

```diff
@@ -36,6 +36,18 @@
                 "galaxy.upload_to_namespace",
                 "ansible.delete_collection",
             ],
+            "galaxy.group_admin": [
+                "galaxy.view_group",
+                "galaxy.delete_group",
+                "galaxy.add_group",
+                "galaxy.change_group",
+            ],
+            "galaxy.user_admin": [
+                "galaxy.view_user",
+                "galaxy.delete_user",
+                "galaxy.add_user",
+                "galaxy.change_user",
```

Comment:
I see `LOCKED_ROLES` only exist under a specific ViewSet in the `VIEWSETS` dictionary. How do these apply to the ViewSet they are under? Once a locked role is in the database, can it be used as a system-wide role that is independent of the ViewSet?

Reply:
Yes. Keeping it with the viewset is just a convenient convention.